Libraries/Lists/VirtualizedList.js

```diff
@@ -1823,7 +1823,7 @@ export default class VirtualizedList extends StateSafePureComponent<
     );
     const item = getItem(data, index);
     const frame =
-      item != null
+      item !== undefined
         ? this._frames[this._keyExtractor(item, index, props)]
         : undefined;
```

Comment:
I looked through this code a bit, and think we should just remove the item check entirely. It looks like there are other places like `pushCells` where we will pass the unprotected item into the keyExtractor already.

Reply:
@NickGerleman  so youre suggesting i remove the item check for frames? 
should i go ahead with it?

Reply:
The check for whether an item is nullish can be removed. Basically always doing L1827.

The check below this change, for whether the frame exists, should be kept.

Reply:
@NickGerleman  ive made those changes, will change the changeLog. please review :) 
please let me know if you think otherwise

Reply:
have updated the changelog